source/MaterialXContrib/MaterialXNode/MaterialXNode.cpp

```diff
@@ -136,20 +136,15 @@ void MaterialXNode::createAndRegisterFragment()
 
         if (_elementPath.length() == 0)
         {
-            // When an empty element path is passed to MaterialXData's
-            // constructor, the first renderable element is selected, which is
-            // a handy feature when creating the node with the command.
-            // However this automatic behavior would complicate state
-            // transitions on attribute edits after the node has been created.
-            // So if the element path attribute is empty, bail early and don't
+            // If the element path attribute is empty, bail early and don't
```

Comment:
Don't need this comment since there is no "hidden" logic in the constructor anymore.